pyclipper/pyclipper.pyx

```diff
@@ -742,9 +745,7 @@ cdef class PyclipperOffset:
         More info: http://www.angusj.com/delphi/clipper/documentation/Docs/Units/ClipperLib/Classes/ClipperOffset/Methods/Constructor.htm
         """
         log_action("Creating an ClipperOffset instance")
-        self.thisptr = new ClipperOffset()
-        self.MiterLimit = miter_limit
-        self.ArcTolerance = arc_tolerance
```

Comment:
I removed these two attributes (MiterLimit and ArcTolerance) because there are already properties defined with the same name, to get and set the respective values.